tools/cellprofiler/enhance_or_suppress_features.xml

```diff
@@ -0,0 +1,253 @@
+<tool id="cp-enhanceorsuppressfeatures" name="EnhanceOrSuppressFeatures" version="@CP_VERSION@">

```

Comment:
Should this be `Enhance Or Suppress Features`?

Can we also use ids like `cp_enhance_or_suppress_features`. Ids are used a lot in configurations on the Admin side so its good to have them readable.

Reply:
i will change the IDs..
but in cellprofiler all modules are named using camel cases.. so maybe keep it so it looks the same as cellprofiler?

Reply:
Your decision :). I think its more readable without... but in the end it counts what users feel confident.

Reply:
ok. I will wait until both of you finish reviewing and then I will start fixing them one by one.

Reply:
no more comments from my side :) Alles super!